### PR TITLE
Fix integration test

### PR DESCRIPTION
**Pull request by fisker**
## Checklist

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -123,7 +123,7 @@
     "esm-utils": "4.0.0",
     "execa": "6.1.0",
     "jest": "28.1.0",
-    "jest-light-runner": "0.2.2",
+    "jest-light-runner": "fisker/jest-light-runner#fix-addSnapshotSerializer",
```
Comment (fisker) [inline package.json:126]: https://github.com/nicolo-ribaudo/jest-light-runner/pull/47

File: tests/integration/__tests__/bundle.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ import * as coreOptions from "../../../src/main/core-options.js";
 import codeSamples from "../../../website/playground/codeSamples.mjs";
 import jestPathSerializer from "../path-serializer.js";
 
-const { require } = createEsmUtils(import.meta);
+const { require, importModule } = createEsmUtils(import.meta);
```
[the remark this answers is not in the document]
Comment (fisker) [reply tests/integration/__tests__/bundle.js:11]: `importModule` = `import(url.pathToFileURL())` on windows the argument must be a URL

[the remark this answers is not in the document]
Comment (sosukesuzuki) [reply tests/integration/__tests__/bundle.js:11]: I got it

**Review by sosukesuzuki — APPROVED** [review]
LGTM, thank you

**Comment by fisker** [comment]
@sosukesuzuki I accidentally changed the fixtures for --cache test, but test still passed https://github.com/prettier/prettier/pull/12859/commits/20d63ecbd2040b1916735da9ef8fcb00f03ca751, also this test changes files and create directories during test, I accidentally committed them several times, I think better use a temp dir to run them, do you think you can do that?

**Pull request merged**